src/social-media-post/models/social-media-post.entity.ts

```diff
@@ -0,0 +1,20 @@
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { UserEntity } from '../../user/models/user.entity';
+
+@Entity('social_posts')
+export class SocialMediaPostEntity {
```

Comment:
A `visits_count` could be a cool feature for measuring how effective is our app at promoting blood donation content.

Reply:
True, that would be nice. I'd prefer to add it as a task to the backlog and work on it eventually as priority for this is low imho but sounds like a cool feature.